app/models/submissions/questions_sub.rb

```diff
@@ -4,6 +4,22 @@
 class QuestionsSub < Submission
   include SubmissionsHelper
   validate :check_time_taken
+
+
+  def grade_question(who_graded, grade_id, message, question, score)
```

Comment:
Changing this to not have an exclamation mark is an incomplete fix: you're still ignoring the success/failure status.